tests/e2e/testkmsv2.bats

```diff
@@ -47,13 +47,10 @@ setup() {
 @test "check keyID hash used for encrypt/decrypt" {
     # expected_hash value is computed based on the key used in CI.
     # this needs to be updated when we rotate that key.
-    local expected_hash="cbda52be2f8c13d323a3b17c4679118a60b91d29454305e02ee485185b6e386f"
-    local metrics=$(kubectl get --raw /metrics)
-
-    hashIDs=$(echo "${metrics}" | grep -oP 'sha256:\K[a-f0-9]+')
-    for hash in ${hashIDs}; do
-        [[ "${hash}" == "${expected_hash}" ]]
-    done
```

Comment:
metrics has multiple hashes (api server identity, keyID and more), so this is not correct. It's just trying to grab all hashes in the output.